src/base/views/_bootstrap-4-screen-sm-min.scss

```diff
@@ -0,0 +1,18 @@
+/*
+  WET-BOEW
+  @title: Bootstrap 4 suplemental style for WET-BOEW - Small view and over (screen only)
+ */
+
+/* Margin */
+.mb-sm-5 {
+	margin-bottom: 50px !important;
```

Comment:
Shouldn't those be in `em` instead of `px`?

Reply:
Bootstrap 4 uses rem.

https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.css

Reply:
> Bootstrap 4 uses `rem`.
> 
> https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.css

We know that Bootstrap 4 uses `rem`, however for minimal impact on overall WxT's layout right now (mostly GCWeb), the community agreed on using `em` temporarily.

Reply:
We will go with px because that is how it was coded in the utility CSS class: https://github.com/wet-boew/wet-boew/blob/master/src/base/proximity/_base.scss

So I am going to change the ```em``` into ```px```.

But in the next version, definitely we should use ```rem```. The challenge of using ```rem``` is we will need to re-caculate the measure for wet-boew + gcweb theme and re-test everything to see if there any visual impact.